packages/components-css/src/index.tsx

```diff
@@ -2,46 +2,46 @@ import '../../../components/Alert/src/index.scss';
+import '../../../components/Card/src/Card/card.scss';
+import '../../../components/Card/src/CardActions/card-actions.scss';
+import '../../../components/Checkbox/src/index.scss';
+import '../../../components/CtaDownload/src/index.scss';
+import '../../../components/CtaEvent/src/index.scss';
+import '../../../components/CtaLink/src/index.scss';
+import '../../../components/DescriptionList/src/index.scss';
 import '../../../components/Divider/src/index.scss';
-import '@gemeente-denhaag/dotindicator/dist/index.scss';
-import '@gemeente-denhaag/eventdate/src/index.scss';
-import '@gemeente-denhaag/formcontrollabel/dist/index.scss';
-import '@gemeente-denhaag/formgroup/dist/index.scss';
-import '@gemeente-denhaag/iconbutton/dist/index.scss';
-import '@gemeente-denhaag/icons/dist/index.scss';
-import '@gemeente-denhaag/image/src/index.scss';
-import '@gemeente-denhaag/link/dist/index.scss';
-import '@gemeente-denhaag/list/dist/list-item.scss';
-import '@gemeente-denhaag/list/dist/list-item-icon.scss';
-import '@gemeente-denhaag/list/dist/list-item-secondary-action.scss';
-import '@gemeente-denhaag/list/dist/list-item-text.scss';
-import '@gemeente-denhaag/list/dist/list-subheader.scss';
-import '@gemeente-denhaag/menu/dist/menu.scss';
-import '@gemeente-denhaag/menu/dist/menu-button.scss';
-import '@gemeente-denhaag/modal/src/index.scss';
-import '@gemeente-denhaag/note/src/index.scss';
-import '@gemeente-denhaag/pagination/src/index.scss';
-import '@gemeente-denhaag/posttypelabel/src/index.scss';
-import '@gemeente-denhaag/pagebuttons/src/index.scss';
-import '@gemeente-denhaag/publicationdate/src/index.scss';
-import '@gemeente-denhaag/tab/dist/styles/tab.scss';
-import '@gemeente-denhaag/tab/dist/styles/tab-panel.scss';
-import '@gemeente-denhaag/tab/dist/styles/tabs.scss';
-import '@gemeente-denhaag/timeline/dist/styles/step.scss';
-import '@gemeente-denhaag/timeline/dist/styles/step-content.scss';
-import '@gemeente-denhaag/timeline/dist/styles/step-icon.scss';
-import '@gemeente-denhaag/timeline/dist/styles/step-label.scss';
-import '@gemeente-denhaag/timeline/dist/styles/timeline.scss';
-import '@gemeente-denhaag/typography/dist/heading.scss';
-import '@gemeente-denhaag/typography/dist/paragraph.scss';
+import '../../../components/DotIndicator/src/index.scss';
+import '../../../components/EventDate/src/index.scss';
+import '../../../components/FormControlLabel/src/index.scss';
+import '../../../components/FormGroup/src/index.scss';
+import '../../../components/IconButton/src/index.scss';
+import '../../../components/Icons/src/index.scss';
+import '../../../components/Image/src/index.scss';
+import '../../../components/Link/src/index.scss';
+import '../../../components/List/src/list-item.scss';
+import '../../../components/List/src/list-item-icon.scss';
+import '../../../components/List/src/list-item-text.scss';
+import '../../../components/List/src/list-subheader.scss';
+import '../../../components/List/src/list-item-secondary-action.scss';
```

Comment:
why did you change the order here? ;)

Reply:
![image](https://user-images.githubusercontent.com/24610692/154441531-3a99f1fc-29e9-4b2f-b116-fed79a318c13.png)

Probably messing around with the wrong IDE shortcuts :P, I'll fix it cheers